README.md

```diff
@@ -94,22 +98,24 @@ pip install -r requirements.txt
     $ python2 pokecli.py -a google -u tejado -p 1234 --location "New York, Washington Square"
 
 ### Bot Configuration via JSON
-    To load arguments for the bot from a JSON file, use the ``--config-json`` argument with the name of a file.
-    Any other command line arguments specified will override the parameters specified in the loaded JSON file.
+To load arguments for the bot from a JSON file, use the ``--config-json`` argument with the name of a file.
+Any other command line arguments specified will override the parameters specified in the loaded JSON file.
 
-    Example - this will load config.json but use cp=1000 and iv=0.7 even if already defined in config.json:
-    $ python2 pokecli.py --config-json config.json -cp 1000 -iv 0.7
+Example - this will load config.json but use cp=1000 and iv=0.7 even if already defined in config.json:
+```
+$ python2 pokecli.py --config-json config.json -cp 1000 -iv 0.7
+```
 
  ### JSON Options
```

Comment:
Space in front needs to be removed since otherwise it doesn't get correctly formatted.